webpack.js

```diff
@@ -15,6 +15,8 @@ webpackConfig.stats = {
 
 const appId = 'assistant'
 webpackConfig.entry = {
+	generationPage: { import: path.join(__dirname, 'src', 'imageGenerationPage.js'), filename: appId + '-imageGenerationPage.js' },
```

Comment:
```suggestion
	imageGenerationPage: { import: path.join(__dirname, 'src', 'imageGenerationPage.js'), filename: appId + '-imageGenerationPage.js' },
```
A little bit of nitpicking :grin: